renderdoc/driver/vulkan/vk_layer.cpp

```diff
@@ -163,7 +163,7 @@ VkResult VKAPI_CALL hooked_vkCreateInstance(const VkInstanceCreateInfo *pCreateI
                                             const VkAllocationCallbacks *pAllocator,
                                             VkInstance *pInstance)
 {
-  WrappedVulkan *core = new WrappedVulkan("");
+  WrappedVulkan *core = new WrappedVulkan(0);
```

Comment:
That's what I don't understand though - the filename is only passed to the Serialiser constructor during replay - and if this function `hooked_vkCreateInstance` is getting called on replay there is a much more serious problem that this is hiding. What is the callstack where it reaches here on replay?